metis-data/metadata-collector
# Update ci to dev env

- test
- remove
- fix
- fix test
- fix
- fix
- fix
- fix default
- fix default
- fix default
- md5 tag
- md5 tag
- later
- deploy dev
- is latest
- is latest
- revert
- revert
- required false
- remove extra )
- remove npm i
- check
- use token
- use token
- fix docker file
- check
- check
- cat
- fix
- unset
- revert NPM_TOKEN_READ_ONLY
- replace npm rc
- latest
- fix
- fix
